src/Propel/Common/Config/Loader/IniFileLoader.php

```diff
@@ -80,6 +80,7 @@ private function parse(array $data): array
 
         foreach ($data as $section => $value) {
             if (is_array($value)) {
+                // @phpstan-ignore-next-line
```

Comment:
We solved this cleaner in current master

Reply:
I'm not sure I understand.  What has been solved in `master`?  I just had a look and it's the same.  The only change here is skipping over the PHPStan error, since it's incorrectly determining the string to be empty here:

`$sections = explode($this->nestSeparator, $section);`

Reply:
No, it does not skip over it.
It uses the proper annotation on it - https://github.com/propelorm/Propel2/blob/master/src/Propel/Common/Config/Loader/IniFileLoader.php#L27